### ISSUE TITLE
Pokemon x and y

### ISSUE THREAD

**Issue by bobbymc1985** [SachinVin/citra_android#166]
I have saw several YouTube videos of pokemon x and y booting up however these were on older devices. I know there is an issue with most devices. Is there around this with settings or is it the case of waiting for an update

**Comment by HelloOO7** [comment]
1. Pokémon X/Y only works with the latest Canary PC builds with some of the LLE changes implemented into the HLE audio core. This build is based of months old code, so don't even bother trying to boot it, as it's pretty much unplayable on PCs with that version.

2. You'll be waiting for an update reaaally long. SachinVin stated they will occassionally push some minor changes, but they'll be busy for the next few months and won't have much time to work on citra-android.

Sources: 

https://github.com/SachinVin/citra_android/issues/164#issuecomment-453852766

https://github.com/SachinVin/citra_android/pull/119#issuecomment-450625269

**Comment by bobbymc1985** [comment]
Yeah thought that. Still great that for example pokemon alpha sapphire works. Looking forward to the next update

**Comment by RedLeo18s** [comment]
@Hello007 i like all of your comments i think your updated to citra emulator and or games.. Do you know how to fix pokemon omega ruby on citra.. I cant play it.. I have the requirements but games didn't work i always finding solution like delete citra and download the latest one delete pokemon omega ruby and download new one but it still not working can you please help me or give me some more details or advises. Please need your help.

**Comment by RedLeo18s** [comment]
I almost forgot my phone is huawei y7 prime.
